src/components/NavGlobal.tsx

```diff
@@ -0,0 +1,50 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { cookies } from 'next/headers';
+
+import cloudGovText from '@/../public/img/logos/cloud-gov-logo-text.svg';
+import cloudGovIcon from '@/../public/img/logos/cloud-gov-icon.svg';
+import dashboardIcon from '@/../public/img/logos/dashboard-icon.svg';
+import cloudPagesIcon from '@/../public/img/logos/cloud-pages-icon.svg';
+
+import { LoginButton } from '@/components/auth/LoginButton';
+import { LogoutButton } from '@/components/auth/LogoutButton';
+
+export function NavGlobal() {
+  const cookieStore = cookies();
+  const authSession = cookieStore.get('authsession');
+
+  return (
+    <div id="nav-global">
+      <div>
+        <div className="nav-global-inner">
+          <Link href="/" className="cloud-logo">
+            <Image
+              id="cloud-gov-icon"
+              unoptimized
+              src={cloudGovIcon}
+              alt="cloud.gov"
+            />
+            <Image
+              id="cloud-gov-logo-text"
+              unoptimized
+              src={cloudGovText}
+              alt="cloud.gov"
+            />
```

Comment:
[blocking request] Similar to @sknep's comment below, VoiceOver Mac is reading "cloud.gov" twice here at desktop widths. Ideally I think we should:
1) on desktop, have a single, combined logo with alt text reading "cloud.gov", then
2) on mobile, hide desktop logo and replace with another image that's the condensed version of the logo (with same alt text) 

@sknep or @beepdotgov Does that sound good, or is there a better approach?

Reply:
I will mess around a little with the single image clipping, but otherwise will switch to displaying a single image for each screen size!

My original instinct was to use CSS to include the images as a `::before` or something, which I still think would be nicer but I got lazy about trying to figure out the asset pipeline...